config/config.go

```diff
@@ -180,13 +180,27 @@ func NewConfig(path string) (*Config, error) {
 		return nil, err
 	}
 
-	err = parseConfig(c.ConfigFile, c.Choria, "", c.rawOpts)
+	err = parseConfig(path, c.Choria, "", c.rawOpts)
 	if err != nil {
 		return nil, err
 	}
 
 	c.parseAllDotCfg()
 
+	if !c.InitiatedByServer {
```

Comment:
There is no way InitiatedByServer can be set at this point since the config was just created in the same function, so we need a rethink, we also need to consider this on the broker to avoid it, so I need to think a bit more about how to approach this.